### DISCUSSION TITLE
REG Hybrid and Flexible Working Section

### DISCUSSION THREAD

**Discussion by edaub** [alan-turing-institute/REG-handbook#146] [Ideas]
I've had several discussions about Hybrid and Flexible working in the past few months and we wanted to capture some of the specifics of what can be supported in the handbook. This discussion is for capturing thoughts of what else might go in this section, where it should be in the handbook, and what level of detail would be acceptable for sharing about particular arrangements given that the handbook is public.

## Things I have discussed that could go in this section

1. Temporary working abroad (@LevanBokeria is collecting details on this)
2. Informal flexible working (i.e. you are free to work late one day, come in late the next on an ad hoc basis without any explicit approval)
3. Formal flexible working (i.e. contractual changes for long term compressed hours, non-standard work hours to support family needs, etc.)
4. Flexibility around travel times when coming to the office (interpreted broadly to include those living in London as well as further away)

Feel free to add other things that you might want included in this section or other thoughts about what would be appropriate for the handbook.

## References

[Institute Flexible Working Policy](https://mathison.turing.ac.uk/page/2216?SearchId=426430)
[Home Working Policy](https://mathison.turing.ac.uk/page/2218)
[Short Term Working Abroad Form](https://mathison.turing.ac.uk/page/2304?SearchId=426436)

**Comment by evelinag** [comment]
I would also like to have a section in the handbook on how we enable hybrid working in terms of team cohesion and culture. This would include the away day(s), hack week and other activities. This feels like a different angle on how we do hybrid working. Should it go into the same section?

**Comment by LevanBokeria** [comment]
Regarding (1) temporary remote work from abroad: [here is the link for the HackMD](https://hackmd.io/@levanbokeria/HJNiTPLc6) where we are collecting information about relevant factors and resources.

**Comment by yongrenjie** [comment]
@edaub: I'm so sorry this is so very late Eric, I've been very distracted over the last couple of weeks, but following on from our discussion I was thinking it would be useful to have a mention of whether it's feasible from both REG and Turing perspectives of working overseas long-term much like @evelinag's arrangement. It's something I've wondered about a couple of times personally and it's also come up in conversation a couple times (in fact, one of those times was on my first day at the Turing!! 😄) so I think would be of general interest to people, even if the answer is no we can't really do this.

Separately from this, I wonder whether it might also be worth specifically touching on specific team events (hack week, away day)? I'm thinking back to 2 or 3 team meetings ago where someone asked whether hack week was compulsory to attend, and of course also some of the discussions we've been having over whether travel / accommodation can be funded from the professional development budget. For people coming from outside of London I think there is often a tradeoff that they have to make, as coming in for team events is really important, but if it comes at a cost of having to spend professional development budget on that, then people may opt to spend it elsewhere. I think it would be useful for these people to know whether e.g. coming in for half of hack week is a plausible / good option.

**Reply by yongrenjie** [reply]
Forgot to say, of course, thank you for putting this together 😊

**Reply by evelinag** [reply]
I'm afraid the answer regarding long-term working overseas is that it is not a standard Turing arrangement, unfortunately. It is fairly complicated in terms of taxes and legal requirements, especially since UK is no longer member of the EU. Happy to talk offline about the details if anyone's interested.

**Reply by martintoreilly** [reply]
On the long-term working overseas question, this won't become a standard offering from Turing. The legal complexities mean that HR will only support it exceptionally and even then only with a very heavy push from that person's Director. There is also an additional complexity that makes this something REG would also find it difficult to support more broadly. The solution we've found with HR is to make the person a contractor rather than an employee and we've recently learned that this makes the person "Directly Incurred" rather than "Directly Allocated" from a UKRI funding perspective. The REG pooled staffing model relies on all / the vast majority of our staff being eligible to be "Directly Allocated" on UKRI grants, as this is how effectively and sustainably cost REG into grants in advance while maintaining flexibility on who actually works on the project.

**Reply by martintoreilly** [reply]
>For people coming from outside of London I think there is often a tradeoff that they have to make, as coming in for team events is really important, but if it comes at a cost of having to spend professional development budget on that, then people may opt to spend it elsewhere. I think it would be useful for these people to know whether e.g. coming in for half of hack week is a plausible / good option.

On this topic, we've carefully considered this on multiple occasions (basically prior to every away day or hack week since lockdowns ended) and the current approach is something we consider a reasonable balance, both in terms of the HR + HMRC constraints and in terms of aligning with fairness considerations from both a REG and wider Turing perspective.

In summary:
- Turing will not pay for travel to London, no matter where people live.
    - This is a firm decision by HR / Turing leadership to not pay for travel into the office, with individual staff needing to factor in funding reasonable travel to the office into their decisions on where to live. This is partly driven by HMRC's view on what qualifies as travel to a "regular place of work" (and therefore not considered a reasonable business expense) and partly by considerations of fairness in terms of not subsidising travel into the office for those who have chosen to live further away (often paying less in housing costs) when not paying for commute costs for those living in / closer to London.

- REG will cover the cost of overnight stays for the team Away Day and Hack Week for people whose journeys are more than 2 hours each way but the costs for this will come out of each individual's nominal professional and development budget (currently £1,000 but we hope to increase this to £1,500 when budgets are confirmed). 
    - Taking these costs from people's nominal personal professional development budgets balances supporting folk who can't travel home each day to choose have the cost of attending away days / hack weeks subsidised by the Turing while not giving them a larger effective per-person budget than folk who live within a daily commute distance of the office. This stems from the same fairness concern feeding into the decision to not pay for travel to London and is part of both ensuring fairness across the team and aligning with the HR / Turing level principles on this.
    - Note that we also ask for a team-level budget to support additional travel, conference fees etc and those who have used up their nominal personal allocation can ask for additional things to be funded from this. However, these requests will be considered alongside other requests and will depend on the overall budget and demand from the team for this. For example, this team pot is used for things such as:
        -  Supporting people attending events they are presenting at where they have exhausted their nominal personal allocation
        - Supporting people going to events REG sees high value in as highly relevant to the profession (e.g. RSECon, similar events for RDS / RCE folk) where they have exhausted their nominal personal allocation
        - Supporting some people each year going to more expensive / further afield events (e.g. in the US).

- Note that, while we don't currently require people to attend the REG Away Day and / or Hack Week in person, recognising that there are many life reasons separate to budget that may make this difficult, we would consider the current arrangements fair and reasonable even if we did require attendance at these events as part of an overall reasonable expectation to occasionally be in the office where it makes a big difference from a business perspective.

**Reply by martintoreilly** [reply]
>I think it would be useful for these people to know whether e.g. coming in for half of hack week is a plausible / good option.

On this specific question, partial attendance at Hack Week is definitely permitted. We don't currently require people to attend the REG Away Day and / or Hack Week in person at all, recognising that there are many life reasons separate to budget that may make this difficult. I think partial attendance would be valuable for team members and people have done this in previous years, so perhaps they can share their experience of this here?

For the Away Day, I think attending both days is likely significantly more useful than attending one day and supporting the costs of an overnight stay should make attending one day vs two days cost neutral for staff at a relatively small fraction of their nominal personal professional development budget.

**Reply by edaub** [reply]
For one data point, I have only been able to come in for the Hack Week for part of the week each time, primarily due to family constraints. I have had mixed success in terms of successfully integrating into a project -- one year I managed to complete a small contribution to one of the projects (done largely during a single day in person), while the other I was less successful in making headway (partially due to needing to swap out my laptop while in the office). Given that my current role on the team is largely managerial, every hour being able to dive into something technical was very much welcome. The social aspects of the Hack Week also made it well worth it for me, even in the case where I didn't manage much of a contribution to a project.
